IMDA-BTG/aiverify-developer-tools
# Synchronize playground widget styles with AI Verify portal styles

Update the themes and styles of the playground to match as closely as possible to the aiverify portal. Fix issue: https://github.com/IMDA-BTG/aiverify-developer-tools/issues/7.

To test:
1) Pull from branch and install
2) npm "ai-verify-plugin playground" on one of the plugins.
3) Check the layout and styles.
